src/plugins/intel_cpu/src/nodes/bin_conv.cpp

```diff
@@ -777,22 +776,22 @@ struct jit_uni_bin_conv_kernel_f32 : public jit_uni_bin_conv_kernel, public jit_
                         auto vmm_dst = Vmm(1 + r * jcp_.ur_w * jcp_.nb_oc_blocking + jj);
 
                         if (isa == x64::avx512_core) {
-                            size_t o_off = 0;
-                            if (jcp_.with_dw_conv) {
-                                o_off = jj * jcp_.oc_block;
-                            } else {
-                                o_off = jj * jcp_.oc * jcp_.ngroups;
-                            }
+                            size_t o_off = [&] {
```

Comment:
Shouldn't it be const btw? Here and in the other similar places.

Reply:
Yes, in many places it can be const. We want to ensure it using [misc-const-correctness](https://clang.llvm.org/extra/clang-tidy/checks/misc/const-correctness.html)